src/components/common/RawDiffLinkButton.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react'
+import styled from '@emotion/styled'
+import { Button as AntdButton } from 'antd'
+import { getDiffURL } from '../../utils'
+
+export const testIDs = {
+  upgradeButton: 'upgradeButton'
+}
+
+const Container = styled.div`
+  display: flex;
+  justify-content: center;
+  height: auto;
+  overflow: hidden;
+  margin-top: 25px;
+`
+
+const Button = styled(AntdButton)`
+  border-radius: 3px;
+`
+
+const RawDiffLinkButton = ({
+  packageName,
+  language,
+  fromVersion,
+  toVersion
+}) => {
+  if (fromVersion === '') {
```

Comment:
I dont know if this is "enough" in terms of checking whether the button should be shown. AFAIK the `fromVersion` and `toVersion` will be always present, so this is protecting only the few cases when the website is being loaded.

Reply:
good point. I think it's enough for now. There is an open issue for displaying "something" when the diff is empty but it shouldn't be addressed here :)